Hrms/src/main/java/Kodlamaio/Hrms/api/controllers/EmployeesController.java

```diff
@@ -0,0 +1,67 @@
+package Kodlamaio.Hrms.api.controllers;
+
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import Kodlamaio.Hrms.business.abstracts.EmployeeService;
+import Kodlamaio.Hrms.entities.concretes.Employee;
+
+@RestController
+@RequestMapping("/api/employees")
+public class EmployeesController {
+
+	private EmployeeService employeeService;
+
+	@Autowired
+	public EmployeesController(EmployeeService employeeService) {
+		super();
+		this.employeeService = employeeService;
+	}
+	
+	@GetMapping("/getall")
+	public List<Employee> getall()
+	{
+		return employeeService.getAll();
+	}
+	
+	@PostMapping("/add")
+	public void add(@RequestBody Employee employee)
+	{
+		employeeService.add(employee);
+	}
+	@DeleteMapping("/delete")
+	public void delete(@RequestBody int id)
+	{
+		employeeService.delete(id);
+	}
+	
+	@GetMapping("/get")
+	public Employee get(@RequestBody @RequestParam int id) {
```

Comment:
@Sadikortaoglan buradaki @RequestParam'ı kullanmıyorsan kaldırmalısın.
```suggestion
	public Employee get(@RequestBody int id) {
```